### PR TITLE
Toggle favorite when click favorite on search screen

### LINKED ISSUE #413: Toggle favorite when click favorite on search screen
**Idea Description**
image

**Reference images and links**

### PR DESCRIPTION
**Pull request by geckour**
## Issue
- close #413

## Overview (Required)
- Connected bookmark icon tap event to `SessionsRepository#onFavoriteToggle`

## Links
- N/A

## Screenrecord
https://user-images.githubusercontent.com/6108088/189922515-5427a529-f170-444d-bbd9-245e5392f5dc.mp4

### CODE REVIEW

**Review by takahirom — APPROVED** [review]
Thanks!

**Comment by takahirom** [comment]
Can you check this? 🙏 
> e: /Users/runner/work/conference-app-2022/conference-app-2022/feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt: (72, 23): Unresolved reference: onFavoriteToggle

File: feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt [hunk 1]
```diff
@@ -60,23 +61,24 @@ import io.github.droidkaigi.confsched2022.ui.UiLoadState.Success
...
+        },
     )
 }
 
 @Composable
 private fun SearchScreen(
     uiModel: SearchUiModel,
     onItemClick: () -> Unit,
-    onBookMarkClick: () -> Unit,
+    onBookMarkClick: (sessionId: TimetableItemId, currentIsFavorite: Boolean) -> Unit,
```
Comment (takahirom) [inline feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:81]: 👍

**Comment by geckour** [comment]
@takahirom 
Sorry for the late response! 🙏 
I may have fixed the issue 👀

**Review by takahirom — APPROVED** [review]
Thanks!

**Pull request merged**